Build/gcc/Makefile.executable_prerequisites

```diff
@@ -79,6 +79,18 @@ HARDWARE_FILE_READER_SOURCES = $(SOURCE_DIR)/OrchBase/HardwareFileReader/Hardwar
                                $(GENERICS_DIR)/uif.cpp \
                                $(HARDWARE_DEPLOYER_SOURCES)
 
+# Sources used by executables that provide POETS name resolution.
+NAME_SERVICES_SOURCES = $(SOURCE_DIR)/NameServer/ABDefs.cpp \
+                        $(SOURCE_DIR)/NameServer/ABRecord.cpp \
+                        $(SOURCE_DIR)/NameServer/ABTask.cpp \
+                        $(SOURCE_DIR)/NameServer/AddressBook.cpp \
+		        $(SOURCE_DIR)/NameServer/SBase.cpp \
+		        $(SOURCE_DIR)/OrchBase/CMsg_p.cpp \
+		        $(SOURCE_DIR)/OrchBase/P_addr.cpp 
+
+# The LogServer message file is a static text file
+LOGSERVER_MESSAGE_FILE_ORIGIN = $(SOURCE_DIR)/OrchestratorMessages.txt
```

Comment:
This looks like something inserted during the merge. DOes Git do this (interleave files?) From reading the documentation it looked ats if it might, which is why I was very VERY reluctant to do a merge, but it looks also as though it does it so subtly that you can't even do a compare to detect differences. If merges are required, on the one hand, and if merge silently interleaves files on the other, and if it doesn't give you an option that allows you to do a line-by-line approval of the merge as it happens, I don't see how it can be possible to pull a branch back into the main line without creating merge issues.

Reply:
This line was introduced by commit `2bbe0a6`, which was before `development` was merged in:

```
commit 2bbe0a6f80930a234fa5d7da04153f833dd9cddd
Author: AlexRast <adr1r17@soton.ac.uk>
Date:   Thu Sep 26 14:48:22 2019 +0100

    re-updated makefiles and message file that were somehow reverted in the last commit
```

Perhaps your issue is with the work prior to this commit.